### DISCUSSION TITLE
F/f not work properly when record macro

### DISCUSSION THREAD

**Discussion by LintaoAmons** [folke/flash.nvim#34] [General]
When I `f"` when was recording a macro and tried to replay it to multiple lines.

It doesn't work properly and the result is not correct.

So maybe automatically disable this plugin when recording a macro?

**Comment by folke** [comment]
Can you show me an example of what exactly went wrong?

Note to self: `vim.fn.reg_recording() ~= ""` can be used to check if we're in a macro

**Reply by LintaoAmons** [reply]
Sorry, Wrong discussion area.... I was supposed to post this in `flash` repo

**Reply by folke** [reply]
moved it

**Comment by LintaoAmons** [comment]
![show](https://github.com/folke/lazy.nvim/assets/95092244/bd1186dc-49b6-488f-8104-55925a1ce421)
When I do `@w` its fine, but if I do `6@w` it not working properly

**Reply by folke** [reply]
Should be fixed now. Would be great if you can confirm!

**Reply by LintaoAmons** [reply]
Smooth and Intuitive!!!
Thx so much to build so many cool stuff~ and make them easy to use
